frontend/packages/dev-console/src/components/user-preferences/SecureRouteFields.tsx

```diff
@@ -171,7 +171,7 @@ const SecureRouteFields: React.FC = () => {
 
   return (
     <div className="pf-v6-c-form">
-      <span className="co-help-text">
+      <span>
```

Comment:
wondering if the form group descriptions should still get subtle styling, per `FormFieldGroupHeader` PatternFly styling: https://www.patternfly.org/components/forms/form#field-groups

Reply:
I am thinking no since we're not using that component and this context is different.  It's really help text here.